Oppdater readme slik at det er lettere å bidra/forstå hva prosjektet er

kharann opened the pull request:
Ta en titt her https://github.com/s0lvang/Karantenehjelpen/blob/project/setup/README.md

Så litt rundt å kokte formatet på ulike readmes

kharann committed:
Add logo and badges to readme

kharann committed:
oppdatering av readme og litt drafting

FredrikAugust committed:
Kommentarer, skriverusk og mindre bilde

Adrianht commented:
Hadde det vært en ide å sette opp enda et firbase prosjekt som er helt åpent? Så har vi fortsatt prod og dev slik vi ønsker det

kharann commented:
> Hadde det vært en ide å sette opp enda et firbase prosjekt som er helt åpent? Så har vi fortsatt prod og dev slik vi ønsker det

Kunne vært en god ide. Har full tillit til at folk ønsker bare å hjelpe🥰

FredrikAugust commented:
Tror kanskje det bare er bedre å gi folk tydelige instruksjoner på hvordan sette opp en egen (?). Hvis ikke ender det fort med at folk bruker opp kvoten på testinstansen - enten med vilje eller ei.

FredrikAugust commented:
Men vi kan jo prøve forsåvidt, og så heller stenge den hvis det går skeis.

kharann committed:
Added some hyperlinks and something about our  techstack

Adrianht committed:
Update README.md

Skriver om teksten litt så det ikke bare angår handleliste + noen mindre endringer

FredrikAugust committed:
Update README.md

Adrianht commented:
Kan vel bruke noe som dette for contributors delen?

https://github.com/sourcerer-io/hall-of-fame

FredrikAugust committed:
La til Sourcerer.io hall of fame

FredrikAugust commented:
La det til nå, men ser ikke noen bilder enda, så mulig det tar litt tid.

Adrianht commented:
> La det til nå, men ser ikke noen bilder enda, så mulig det tar litt tid.

Den har dukket opp nå 🥳

kharann commented:
Er kanskje mulig å vise bilde av ALLE contributors, og ikke bare top 7?

EDIT: Nvrmind leste innpå siden, den tar kun opp til 7.

aslakhol committed:
add history to readme

aslakhol committed:
Update README.md

aslakhol committed:
Update README.md

aslakhol commented:
Har lagt inn historie, og referanse til tipsene fra frivillighetssentralen. Det her ser dritbra ut! Få det inn 🚀

kharann committed:
Add an example .env file

asd

kharann committed:
Update README.md

kharann committed:
Update README.md

kharann committed:
Update README.md

FredrikAugust committed:
Merge branch 'master' into project/setup

FredrikAugust commented:
@kharann klar til å merges?

Adrianht commented:
Linken til logoen må vel oppdaters @plusk ?

plusk committed:
Correct logo attribution

kharann commented:
Jeg er ganske fornøyd :)), må bare åpne pull requested og kjøre en read gjennom hele greiene

aslakhol committed:
fix capitalization
